### PR TITLE
Add support for more SLES versions

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

**Comment by anders-larsson** [comment]
FYI: SLES 12 files probably needs updating if they should reflect system defaults. They're currently just copy-pasted from earlier version

File: data/os/SLES/12.3.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+postfix::main_daemon_directory: '/usr/lib/postfix'
```
Comment (Phil-Friderici) [inline data/os/SLES/12.3.yaml:2]: shouldn't it be `/usr/lib/postfix/bin` for >= 12.3 ?
If so, we wouldn't the config files for 12.3 & 12.4 as the needed params would come from 12.yaml instead.

Comment (anders-larsson) [reply data/os/SLES/12.3.yaml:2]: Indeed, they need verification.

**Comment by Phil-Friderici** [comment]
> FYI: SLES 12 files probably needs updating if they should reflect system defaults. They're currently just copy-pasted from earlier version

Ok, will do so and only take over 15.yaml instead.

File: data/os/SLES/12.3.yaml (hunk 1 above)
Comment (anders-larsson) [reply data/os/SLES/12.3.yaml:2]: Could it be that I copied 12.yaml instead of 12.2?

**Comment by anders-larsson** [comment]
> > FYI: SLES 12 files probably needs updating if they should reflect system defaults. They're currently just copy-pasted from earlier version
> 
> Ok, will do so and only take over 15.yaml instead.

We're probably going to need to go through all SLES15 since we know at least SLES 15.2 and SLES15.3 differs (if I remember correctly).

File: data/os/SLES/12.3.yaml (hunk 1 above)
Comment (Phil-Friderici) [reply data/os/SLES/12.3.yaml:2]: looks to me that you copied 12.0 instead of 12 ;)

Comment (anders-larsson) [reply data/os/SLES/12.3.yaml:2]: Ah. Good times.

**Pull request closed without merging**